content/news/XXXX-XX-XX-jack-zero-latency.md

```diff
@@ -0,0 +1,44 @@
+title: "Has JACK zero latency?"
+authors: Daniel Schürmann
+tags: 2.3, jack
+date: 2021.03.18 02:42:18
+
+In regular intervals, we discuss how much latency the JACK layer introduces. The [JACK FAQ](https://jackaudio.org/faq/no_extra_latency.html) state that:
```

Comment:
Let's give some more context to readers that aren't experts in the field. E. g.:
```suggestion
In regular intervals, we discuss how much latency the JACK Audio Connection Kit introduces. That is one of the [Sound APIs](https://manual.mixxx.org/2.3/en/chapters/preferences.html#sound-api) that Mixxx supports on Linux and it's a layer on top of the Advances Linux Sound Architecture (ALSA). The [JACK FAQ](https://jackaudio.org/faq/no_extra_latency.html) state that:
```